js/components/TelegramInstructions.vue

```diff
@@ -1,13 +1,25 @@
 <template>
 	<div>
 		<p>
-			In order to receive authentication codes via Telegram, you first
-			have to start a new chat with the bot set up by your admin.<br>
-			Secondly, you have to obtain your Telegram ID via the <a
-				href="https://telegram.me/get_id_bot"
-				target="_blank"
-				rel="noreferrer noopener">ID Bot</a>.
-			Enter this ID to receive your verification code below.
+			In order to receive authentication codes via Telegram, your admin needs
+			to setup a Telegram Bot that will send the authentication codes to you.
+			Once this has been done:<br>
+			<ol>
+			  <li>
+					Start a conversation with the Bot. You can ask your admin for
+					the bot name.
+				</li>
+				<li>
+					You need your Chat Id. You can get it sending a message to the <a
```

Comment:
I don't think this is correct. What the user enter is their *user ID*. See #110

Reply:
Do you agree to include a more detailed description of Chat Id and Group Id based on the comment [430174531](https://github.com/nextcloud/twofactor_gateway/issues/110#issuecomment-430174531)?

I can rephrase it to make it sorter

Reply:
In the light of our findings in #110 I think the whole setup process should change a bit and remove the current (and buggy) setup logic to just ask the user for their user id. I will look into this soonish.